adagrams/game.py

```diff
@@ -1,11 +1,144 @@
+import random
+
 def draw_letters():
-    pass
+    letters_list = []
+    pool_of_letters_dict = {
+        "A": 9,
+        "B": 2,
+        "C": 2,
+        "D": 4,
+        "E": 12,
+        "F": 2,
+        "G": 3,
+        "H": 2,
+        "I": 9,
+        "J": 1,
+        "K": 1,
+        "L": 4,
+        "M": 2,
+        "N": 6,
+        "O": 8,
+        "P": 2,
+        "Q": 1,
+        "R": 6,
+        "S": 4,
+        "T": 6,
+        "U": 4,
+        "V": 2,
+        "W": 2,
+        "X": 1,
+        "Y": 2,
+        "Z": 1
+
+    }
```

Comment:
With a variable like LETTER_POOL which is a constant, it's just slightly more pythonic to move it outside the function and make it a global constant. The only thing this would change is the need to make a copy of the dictionary inside the function.